components/tabs/examples/tabs_sync.rs

```diff
@@ -10,7 +10,7 @@ use cli_support::prompt::prompt_char;
 use clipboard::{ClipboardContext, ClipboardProvider};
 use tabs::{RemoteTab, TabsEngine};
 
-type Result<T> = std::result::Result<T, failure::Error>;
+type Result<T> = std::result::Result<T, anyhow::Error>;
```

Comment:
In general this and a few other places should just be `use anyhow::Result`.

Note that anyhow::Result does the trick where you can do `Result<T>` for `Result<T, anyhow::Error>` while letting Result<T, E> still work (this code predates that trick working at all, though)